client/my-sites/post-type-list/post-actions-ellipsis-menu/wp-block-actions-ellipsis-menu.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import PostActionsEllipsisMenu from './';
+import PostActionsEllipsisMenuEdit from './edit';
+import PostActionsEllipsisMenuTrash from './trash';
+
+export class WpBlockActionsEllipsisMenu extends Component {
```

Comment:
Optional, but this is simple and short enough for a functional component if you'd like